catalyst_voices/README.md

```diff
@@ -97,6 +97,31 @@ you can use the following command:
 flutter build web --target apps/voices/lib/configs/main_web.dart --dart-define SENTRY_DSN=REPLACE_WITH_SENTRY_DSN_URL
 ```
 
+### Code Generation
+
+In some section of this repo we use code generation for generation:
+
+* Catalyst Gateway OpenAPI
+* localization files
+* assets files
+* routes files for navigation
+
+To generate code run in root directory:
+
+```sh
+earthly ./catalyst_voices+code-generator 
+```
+
+To save generated code locally run in root directory:
+
+```sh
+earthly ./catalyst_voices+code-generator --save_locally=true
+```
+
+Keep in mind that You will need GITHUB_TOKEN to be able to run this earthly target.
+In root directory there is a template file `.secret.template` save it as `.secret` and fill in the GITHUB_TOKEN.
+This file should be ignored by git, but make this is the case.
+
```

Comment:
```suggestion
### Code Generation

This project utilizes automatic code generation for the following components:
- Catalyst Gateway OpenAPI
- Localization files
- Asset files
- Navigation route files

#### Running Code Generation

 **Basic Generation**
 
To generate code, run the following command in the root directory:

`earthly ./catalyst_voices+code-generator`

**Local Saving**

To save the generated code locally, use the `--save_locally` flag:

`earthly ./catalyst_voices+code-generator --save_locally=true`

#### GitHub Token / PAT Setup

 **‼️Important**

A valid `GITHUB_TOKEN`/ `PAT` is required to run the earthly target.

**Token Configuration:**
1. Locate the `.secret.template` file in the root directory
2. Create a copy of this file and name it `.secret`
3. Add your `GITHUB_TOKEN` to the `.secret` file

#### Security Notes
- The `.secret` file should be included in `.gitignore`
- Verify that git does not track the `.secret` file before committing
```

Reply:
@LynxLynxx, Please update the Table of Contents in the README file.

Reply:
Good suggestion for better readability! Thanks!